app/services/coupons/update_service.rb

```diff
@@ -30,7 +41,7 @@ def update(args)
     end
 
     def update_from_api(organization:, code:, params:)
```

Comment:
It might be done in a different PR, but I think we should merge the two `create` / `create_from_api` methods, as we have done for `Plans::UpdateService` for example.

Reply:
Created a card for that